site/content/docs/deployment-projects/cdk-project.md

```diff
@@ -0,0 +1,108 @@
+# CDK Project
+
+A custom deployment project uses a C# Cloud Development Kit (CDK) project to create the AWS infrastructure as a CloudFormation stack based on the settings provided by the user and to deploy your project to that infrastructure.
+
+
+### CDK Refresher
+The following links are useful resources to learn how to write .NET CDK projects.
+
+* [.NET CDK Workshop](https://cdkworkshop.com/40-dotnet.html)
+* [.NET CDK Reference](https://docs.aws.amazon.com/cdk/api/v2/dotnet/api/index.html)
+* [AWS CDK Developer Guide](https://docs.aws.amazon.com/cdk/v2/guide/home.html)
+
+The main concepts to understand about the CDK are the following.
+
+* Stack - this is the top level container for all of the AWS resources represented as constructs. In deployment projects this type is usually the `AppStack` type in the project.
+* Constructs - AWS resources are modeled as constructs. Some constructs are a one-to-one mapping to a single AWS resource. The CDK also has higher level abstractions where a single construct represents a collection of AWS resources that solve a specific problem.
```

Comment:
nit: Replace "specific" with "common". Makes it sounds more useful

Reply:
Addressed in new commit.